tests/foreman/cli/test_contenthost.py

```diff
@@ -567,115 +559,38 @@ def setUp(self):
 
```

Comment:
If I understand it right, we need to move the vm code from setup to individual tests to handle:
```py
self.vm = VirtualMachine(distro='rhel71')
self.vm.create()
self.vm.install_katello_cert()
self.vm.register_contenthost(
self.activation_key['name'], self.org['label']
)
# Install katello-agent
self.vm.enable_repo(REPOS['rhst7']['id'])
self.vm.install_katello_agent()
```
If we move this code to the tests, can we reuse setup?  By this way, tests can create their own vms, but use all the other pre-requisites from existing org, ak, content-view, publish, promote, etc?